### PR TITLE
AccessKey is base64 encoded

### LINKED ISSUE #31: Read in env file as base64 encoded
We chose to base64 encode all environment variables. So we need to update the `BastTest` class for integration test so that it should be able to decode base64 string.

### PR DESCRIPTION
**Pull request by lf-jason-jia**
Fix #31 (See the linked issue for details)

### CODE REVIEW

**Review by steven-xu-lf — COMMENTED** [review]
was the github secret access key already changed to base64?

**Comment by lf-jason-jia** [comment]
Not yet, I will change it soon.

**Comment by lf-jason-jia** [comment]
> was the github secret access key already changed to base64?

I guess may be I should do it now since otherwise the integration tests may fail to run.

**Comment by steven-xu-lf** [comment]
> Not yet, I will change it soo

> > was the github secret access key already changed to base64?
> 
> I guess may be I should do it now since otherwise the integration tests may fail to run.

yea i was wondering how the integration tests for this PR were still able to pass after you added that change

**Comment by lf-jason-jia** [comment]
> yea i was wondering how the integration tests for this PR were still able to pass after you added that change

Oh, because it's configured to try github secrets first and if there's nothing there, it will read from the file. The one on github is still not base64 encoded so it works fine. I only change the path that loads from the file because there's no quotation escaping issue for github secrets.

**Comment by lf-jason-jia** [comment]
But I guess we need to change that part too so that things stay consistent.

**Comment by lf-jason-jia** [comment]
Hi @sxu1998 , I updated the keepass file and the code so that it also decodes from github secret.

File: tests/integration/BaseTest.cs [hunk 1]
```diff
@@ -37,10 +48,26 @@ private static void TryLoadFromDotEnv(string fileName)
...
+            }
+        }
+
         private void PopulateFromEnv()
         {
-            ServicePrincipalKey = Environment.GetEnvironmentVariable("DEV_CA_PUBLIC_USE_TESTOAUTHSERVICEPRINCIPAL_SERVICE_PRINCIPAL_KEY");
-            AccessKey = JsonConvert.DeserializeObject<AccessKey>(Environment.GetEnvironmentVariable("DEV_CA_PUBLIC_USE_INTEGRATION_TEST_ACCESS_KEY"));
+            ServicePrincipalKey = Environment.GetEnvironmentVariable(SpKeyVar);
+            
+            if (!HasBase64Decoded)
```
Comment (jason-chan-lf) [inline tests/integration/BaseTest.cs]: Could we just decode the base64 access key here instead? And then use it directly to deserialize it into the AccessKey object without saving it back to the env?

Comment (lf-jason-jia) [reply tests/integration/BaseTest.cs]: Yeah, let me update the code.

**Review by jason-chan-lf — APPROVED** [review]

**Pull request merged**